src/TypeScript.UnitTests/Analyzer/EslintBridgeAnalyzerTests.cs

```diff
@@ -272,7 +272,9 @@ public void Dispose_UnregisterFromConfigChangedEvent()
         }
 
         [TestMethod]
-        public async Task OnSolutionChanged_NextAnalysisCallsInitLinter()
+        [DataRow(true, false)]
+        [DataRow(false, true)]
+        public async Task OnSolutionChanged_NextAnalysisCallsInitLinter(bool isSolutionOpen, bool shouldReinitLinter)
```

Comment:
I could have just used one variable since the expected behaviour is the reverse of "isSolutionOpen", but I think the version is more declarative.

Reply:
Similar to the test below -- the name of this test is now misleading. It says "NextAnalysisCallsInitLinter" but it's not true.
I think it's worth splitting it to 2 tests, despite the copy-paste.

Reply:
Having renamed the tests, it's much more obvious what the expected behaviour should be from the value of the "solution open" property, so I've dropped the second parameter.